infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LogReplicationServer.java

```diff
@@ -104,6 +112,14 @@ private void createSinkManagers(String localEndpoint, ServerContext serverContex
         }
     }
 
+    private LogReplicationSinkManager createSinkManager(LogReplicationSession session) {
+        LogReplicationSinkManager sinkManager = new LogReplicationSinkManager(localEndpoint,
+                sessionManager.getMetadataManager(), serverContext, session, sessionManager.getContext(session));
```

Comment:
not sure about java coding convention. But the following method structure is more readable

```
func(param1,
        param2,
        param3,
        ....);
```

Feel free to ignore it if this practice is not followed here.

Reply:
I have not seen this practice in corfu. So leaving it as is.